lms/djangoapps/verify_student/models.py

```diff
@@ -1180,6 +1237,9 @@ def get_location_id(cls, photo_verification):
             return ""
 
 
+# DEPRECATED: this feature has been permanently enabled.
+# Once the application code has been updated in production,
+# this table can be safely deleted.
```

Comment:
Minor: It might be a good idea to create a dev story to remind us about this cleanup work. If you do, I'd include the ticket number in this comment.